Modules/_sqlite/prepare_protocol.c

```diff
@@ -30,54 +30,33 @@ int pysqlite_prepare_protocol_init(pysqlite_PrepareProtocol* self, PyObject* arg
 
 void pysqlite_prepare_protocol_dealloc(pysqlite_PrepareProtocol* self)
 {
-    Py_TYPE(self)->tp_free((PyObject*)self);
+    PyTypeObject *tp = Py_TYPE(self);
+
```

Comment:
nitpick: would you mind to remove this empty line?

Reply:
I can do that, but isn't that against [PEP 7](https://www.python.org/dev/peps/pep-0007/#code-lay-out) recommendations?

Reply:
Oh, I never noticed "Function definition style: function name in column 1, outermost curly braces in column 1, blank line after local variable declarations." sentence of the PEP 7. The Python code base is far from respecting that :-/

Reply:
Well, the `_sqlite3` module is far from PEP 7 anyways :(